src/client/dfuse/dfuse_core.c

```diff
@@ -653,7 +655,7 @@ dfuse_set_default_cont_cache_values(struct dfuse_cont *dfc)
 	dfc->dfc_dentry_timeout     = 1;
 	dfc->dfc_dentry_dir_timeout = 5;
 	dfc->dfc_ndentry_timeout    = 1;
-	dfc->dfc_data_caching       = true;
+	dfc->dfc_data_timeout       = 60 * 10;
```

Comment:
is there some guidance from somewhere that 10 mins is best for default?

Reply:
No, that's something we need to work out through benchmarking.